app/utils/jspsych/functions.js

```diff
@@ -12,6 +13,22 @@ import {
   ExperimentParameters
 } from '../../constants/interfaces';
 
+// create and open an outlet to stream data
+const info = lsl.create_streaminfo(
+  'Dummy',
+  'Marker', // type
+  1, // number of channels
+  10, // sample rate in 1 sec.
```

Comment:
based on this [example](https://github.com/labstreaminglayer/liblsl-Python/blob/master/pylsl/examples/SendStringMarkers.py), I would suggest `10` -> `0`.

Reply:
Thank you for pointing out to this example.